frontend/src/components/pages/student/StudentAssessment.tsx

```diff
@@ -40,33 +41,21 @@ const AssessmentSummary = (): React.ReactElement => {
             {assessmentMetadata.startTime}
           </Text>
           <SimpleGrid columns={{ base: 1, md: 2 }} gap={14} pt="2em">
-            <Box
-              backgroundColor="rgba(232, 237, 241, 0.2)"
-              borderRadius="10px"
-              marginTop="7%"
-              padding="2em"
-            >
-              <Text color="blue.300" mb="2em" textStyle="subtitle2">
-                Assessment Question Summary
-              </Text>
-              <HStack justifyContent="space-between">
-                {/* TODO: switch to smallerParagraph after Cindy merges her PR */}
-                <Text textStyle="paragraph">Number of Questions:</Text>
-                <Text textStyle="paragraph">
-                  {assessmentMetadata.numOfQuestions}
-                </Text>
-              </HStack>
-              <HStack justifyContent="space-between">
-                <Text textStyle="paragraph">Total Number of Points:</Text>
-                <Text textStyle="paragraph">
-                  {assessmentMetadata.totalPoints}
-                </Text>
-              </HStack>
-              <br />
-              <Text textStyle="paragraph">Question Types:</Text>
-              <QuestionTypeImages
-                questionTypes={assessmentMetadata.questionTypes}
-              />
+            <Box marginTop="7%">
```

Comment:
can we remove this outer `<Box>` and increase pt="3em" in line 43? and also remove marginTop="7%" from line 63